packages/@secretlint/types/vitest.config.ts

```diff
@@ -0,0 +1,10 @@
+import { defineConfig } from "vitest/config";
+
+export default defineConfig({
+    test: {
+        typecheck: {
+            enabled: true,
+            include: ["test/**/*.test-d.ts"],
```

Comment:
To draw your attention, current setup does not work. Try this:

```suggestion
            include: ["test/**/*.test-d.ts"],
            tsconfig: "./tsconfig.test.json"
```

---

Or open the test file in IDE and you will see an error:

<img width="837" height="225" alt="Image" src="https://github.com/user-attachments/assets/516e3d47-d1cf-4fcd-9465-1699340970d1" />

Vitest does not see this problem, because of instead `tsconfig.test.json` it uses `tsconfig.json` that does not include the test files. In other words, it runs `tsc --noEmit -p tsconfig.json` instead of `tsc --noEmit -p tsconfig.test.json`.

The alarming part for me is that a test file name is actually included in the report.